### DISCUSSION TITLE
Intellibrite is Back

### DISCUSSION THREAD

**Discussion by arrmo** [rstrouse/nodejs-poolController-dashPanel#67] [General]
FYI, seeing this "again" (not sure it ever went away ... LOL),
![image](https://user-images.githubusercontent.com/8756384/208280224-8a8674a7-98cc-4a7c-8cab-764b1f6505e7.png)

And just a thought, but - why are Pool and Spa Lights not in the Lights section? That may be on me 🤣 

Thanks!

**Comment by rstrouse** [comment]
They are not in the lights section because IntelliTouch is reporting them as a feature.  If you go into the configuration and uncheck Show As Feature they will pop down to the lights section.  I assume you do not have any color changing lights but IntelliTouch is reporting the IntelliBrite messages.  I think it has to do with the fact that the IntelliBrite controller exists no matter what on the OCP.  Give me a replay so I can see what is in your config.

**Reply by arrmo** [reply]
> They are not in the lights section because IntelliTouch is reporting them as a feature. If you go into the configuration and uncheck Show As Feature they will pop down to the lights section.

Makes sense, thanks!



> I assume you do not have any color changing lights but IntelliTouch is reporting the IntelliBrite messages. I think it has to do with the fact that the IntelliBrite controller exists no matter what on the OCP. Give me a replay so I can see what is in your config.

I do not 😆. Sure you bet - did a capture, reload the config ... this help? Thanks again.
[replay.zip](https://github.com/rstrouse/nodejs-poolController-dashPanel/files/10264251/replay.zip)

**Reply by rstrouse** [reply]
I am confused by this.  The replay above is for a Nixie single body not an IntelliTouch i7+3.  Did you upload the wrong files?

**Comment by arrmo** [comment]
I don't think so 😜. I saw that too, so just redid the capture - very odd, but it showed i7+3, but after the capture (and Capture Configuration Reload) ... it seemed to change, to Nixie. So I restarted njsPC, and it's back to i7+3 again. Huh?

Make any sense?

Thanks!

**Reply by rstrouse** [reply]
Did you wait until it finished reloading the panel?

**Reply by arrmo** [reply]
I did 😆. Just tried again, and it almost immediately goes from Initializing to Ready, unlike before => and it (third time) changed from i7+3 to Nixie Single Body. Then ... restart njsPC, and back to i7+3.

Thinking it's hitting some bug that is causing this? FYI, I'm on v8.0.0.

Thanks!

**Reply by rstrouse** [reply]
Are you connected using ScreenLogic?

**Reply by arrmo** [reply]
Nope - just my (old) OCP and ICP. And of course njsPC (2 of them 😆).
